### PR TITLE
Now we can get all the srtm1/3 data for NZ...

### PR DESCRIPTION
**Pull request by khandieyea**
Please don't merge, see comment below

### CODE REVIEW

File: create_output.py [hunk 1]
```diff
@@ -63,19 +63,19 @@ def create(file_name, receive_sensitivity, definition):
         print 'Failed converting ppm to png'
     try:
         print 'Converting white to transparency'
-        #os.system(w2t)
+        os.system(w2t)
```
Comment (khandieyea) [inline create_output.py]: You've had this commented out @gotofftherails, can you tell me why? Do they not work?

Comment (gotofftherails) [inline create_output.py]: I think it was misbehaving from memory.  With some particular files.

**Comment by gotofftherails** [comment]
Hey...one thing I am looking at doing is incorporating the whole of NZ 8m Digital Elevation Model from LINZ.

https://data.linz.govt.nz/layer/1768-nz-8m-digital-elevation-model-2012/

**Comment by khandieyea** [comment]
Yea that'd be great

**Pull request closed without merging**